src/summary/history/allergy-card.component.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import { performPatientAllergySearch } from "./allergy-intolerance.resource";
+import style from "./allergy-card-style.css";
+
+export default function AllergyCard(props: AllergyCardProps) {
+  const [patientAllergy, setPatientAllergy] = React.useState(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
```

Comment:
As described in previous comments, you should return a function from your effect that cancels the network request:

```
return () => {
  abortController.abort()
}
```